_sass/data.scss

```diff
@@ -1,16 +1,62 @@
+.visualization {
+  display: flex;
+  flex-direction: column;
+
+  &__data {
+    display: flex;
+    flex-direction: row;
+  }
+}
+
 path {
   stroke: #333;
   stroke-width: .5;
   stroke-linejoin: round;
   stroke-linecap: round;
 }
 
+.sidebar {
+  border: 1px solid #707070;
+  border-radius: 5px;
+  box-shadow: 0 3px 20px rgba(0,0,0,.16);
+  display: flex;
+  flex-direction: column;
+  padding: 25px;
+  width: 535px;
```

Comment:
Would you consider converting the pixel values to `rem` (just px divded by 16). It tends to make it easier for the page to scale if you jump the font size (see https://engageinteractive.co.uk/blog/em-vs-rem-vs-px)

Reply:
Done! There's a few places I left the pixel values, namely border radii, box shadow, and the size of the gold icons. I need to revisit naming the icon class and can convert those to `rem` then, but that's going to be a wee bit more of an undertaking.